platform/core/src/utils/combineFrameInstance.ts

```diff
@@ -73,26 +67,65 @@ const combineFrameInstance = (frame, instance) => {
         ImagePositionPatientToUse = [position[0], position[1], position[2]];
       }
     }
+    const sharedInstance = createCombinedValue(instance, SharedFunctionalGroupsSequence?.[0]);
+    const newInstance = createCombinedValue(
+      sharedInstance,
+      PerFrameFunctionalGroupsSequence?.[frameNumber]
+    );
 
-    const newInstance = Object.assign(instance, { frameNumber: frameNumber });
-
-    // merge the shared first then the per frame to override
-    [...shared, ...perFrame].forEach(item => {
-      Object.entries(item).forEach(([key, value]) => {
-        newInstance[key] = value;
-      });
+    // Note: do not assign directly to newInstance.ImagePositionPatient
```

Comment:
This note doesn't seem to apply - I tried with:
objPar = { a: 3, b:4 }
objChild = Object.create(objPar)
objChild.a = 'ay'
objChild.c = 'c'
console.log("obj=", objChild.a, objChild.b, objChild.c, objPar.a)
And I see ay, 4, 'c', 3
which is what I expect.

Reply:
I wonder if the instance is a proxy object and what we are seeing is a proxy assignment?

Reply:
I don't know what is happening but it is the issue. The `instance.ImagePositionPatient` should not change